app/models/organizations/business_line.rb

```diff
@@ -4,4 +4,256 @@ class BusinessLine < Organization
   def tasks_url
     "/decision_reviews/#{url}"
   end
+
+  def in_progress_tasks(
```

Comment:
```suggestion
  # :reek:LongParameterList
  def in_progress_tasks(
```

Since we'll want the default param values I can't think of a better way to pass in all of the requisite information to the task methods, so disabling the 3+ param check seems reasonable in this case. Do you have any ideas on how we might address the reek lint warning?